substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/Range.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020 Red Hat, Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.oracle.objectfile.elf.dwarf;
+
+/**
+ * details of a specific address range in a compiled method
+ * either a primary range identifying a whole method
+ * or a sub-range identifying a sequence of
+ * instructions that belong to an inlined method
+ */
+
+public class Range {
```

Comment:
I did not see the code where your elf.darf entities get transformed into their binary representations. Do I miss something or is this simply not ready yet?

Reply:
Sorry for the noise. Found it in `DwarfSections.java` (github did not show me that file first because it too large ... with it really is ;-) )

Reply:
Yes, I will split out the embedded classes so the code can be in separate files.